### PR TITLE
Fix #325: Added unit test for showMediaLabel param

### LINKED ISSUE #325: Unit Test that the showMediaLabel parameter of the FeedItem is reflected
## Kind (Required)

- Improvement

 ## Overview (Required)

- The title says everything
image

 ## Links	

- test sample : #310

### PR DESCRIPTION
**Pull request by anandwana001**
## Issue
- close #325

## Overview (Required)
- Unit test for showMediaLabel parameter
- Added Two test for both the cases below mentioned: 
   - showMediaLabel - true, the media text should be visible
   - showMediaLabel - false, the media should not be visible

## Screenshot
Screenshot 2021-03-09 at 13 00 22

### CODE REVIEW

**Comment by anandwana001** [comment]
@takahirom PTAL

**Comment by takahirom** [comment]
👀

File: uicomponent-compose/feed/src/androidTest/java/io/github/droidkaigi/feeder/feed/FeedItemTest.kt [hunk 1]
```diff
@@ -34,4 +35,42 @@ class FeedItemTest {
...
+                    feedItem = feedItem,
+                    favorited = true,
+                    showMediaLabel = true,
+                    onClick = { },
+                    onFavoriteChange = { }
+                )
+            }
+        }
+
+        composeTestRule.onNodeWithTag("MediaLabel", useUnmergedTree = true).assertIsDisplayed()
```
Comment (takahirom) [inline uicomponent-compose/feed/src/androidTest/java/io/github/droidkaigi/feeder/feed/FeedItemTest.kt:55]: I didn't know this option 👍

**Review by takahirom — APPROVED** [review]
LGTM!

**Pull request merged**